handler/src/main/java/io/netty/handler/ssl/OpenSslContext.java

```diff
@@ -301,6 +312,14 @@ public final OpenSslSessionStats stats() {
         return sessionContext().stats();
     }
 
+    /**
+     * Specify if remote initiated renegation is supported or not. If not supported and the remote side tries
+     * to initiate a renegation a {@link SSLHandshakeException} will be thrown during decoding.
+     */
+    public void setRejectRemoteInitiatedRenegation(boolean rejectRemoteInitiatedRenegation) {
```

Comment:
@Scottmitch should this me also "rejectRemoteInitiatedRenegotiation" or is "rejectRemoteInitiatedRenegation" right ? You are the native speaker ;)